### ISSUE TITLE
Experiment hyperparameters

### ISSUE THREAD

**Issue by NoName5990** [BIT-DA/Annotator#2]
Hi, @BinhuiXie, thank you for your excellent work. I would like to know your experiment details about hyperparameters in source-only and target-only. eg: momentum and weight decay of SGD, T-max of consinedecayLR, and if using data augment during the training phase. Additionally, if you can impart more information about source-only or target-only to me, I will be particularly grateful to you.

**Comment by BinhuiXie** [comment]
Code released. Thanks.
